### ISSUE TITLE
SvelteKit node adapter  started failing after latest update

### ISSUE THREAD

**Issue by lovasoa** [sveltejs/kit#2098]
### Describe the bug

After the latest update, SvelteKit node adapter started failing with:

```
> Using @sveltejs/adapter-node
 > .svelte-kit/node/intermediate/index.js:1:29: error: Could not resolve "../output/server/app.js"
    1 │ import { init, render } from '../output/server/app.js';
      ╵                              ~~~~~~~~~~~~~~~~~~~~~~~~~

> Build failed with 1 error:
.svelte-kit/node/intermediate/index.js:1:29: error: Could not resolve "../output/server/app.js"
Error: Build failed with 1 error:
.svelte-kit/node/intermediate/index.js:1:29: error: Could not resolve "../output/server/app.js"
    at failureErrorWithLog (/root/sanipasse/node_modules/esbuild/lib/main.js:1449:15)
    at /root/sanipasse/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/root/sanipasse/node_modules/esbuild/lib/main.js:1049:65)
    at buildResponseToResult (/root/sanipasse/node_modules/esbuild/lib/main.js:1129:7)
    at /root/sanipasse/node_modules/esbuild/lib/main.js:1236:14
    at /root/sanipasse/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/root/sanipasse/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/root/sanipasse/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (node:events:394:28)
    at Socket.emit (node:domain:475:12)

``` 

### Reproduction

I ran npm update and npm audit fix --force in sanipasse: https://github.com/lovasoa/sanipasse

### System Info

```shell
npx envinfo --system --binaries --browsers --npmPackages "{svelte,@sveltejs/*,vite}"

  System:
    OS: Linux 5.13 Fedora 34 (Workstation Edition) 34 (Workstation Edition)
    CPU: (8) x64 Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz
    Memory: 2.13 GB / 15.28 GB
    Container: Yes
    Shell: 5.1.0 - /bin/bash
  Binaries:
    Node: 16.4.0 - /usr/bin/node
    Yarn: 1.22.10 - /usr/bin/yarn
    npm: 7.18.1 - /usr/bin/npm
  Browsers:
    Firefox: 90.0.2
  npmPackages:
    @sveltejs/adapter-node: next => 1.0.0-next.37 
    @sveltejs/adapter-static: next => 1.0.0-next.15 
    @sveltejs/kit: next => 1.0.0-next.142 
    svelte: ^3.38.3 => 3.42.1
```

### Severity

blocking all usage of SvelteKit

**Comment by lovasoa** [comment]
Fixed by : https://github.com/sveltejs/kit/commit/a12beb04a4075c8277b857f96860e7ccdb3daa86#diff-162e563a978cd94e14ce386aabc0b584dec78659ec3d654e28bf832e5fd854ef

Maybe there could be some automated or manual testing before pushing to npm ? It looks like something that could be caught in CI...

**Comment by Conduitry** [comment]
Closing as this was indeed fixed by #2096 and was published to npm.

There's not currently a good way to test adapters in CI. This is a known TODO.
